cmd/minishift/cmd/start_preflight.go

```diff
@@ -44,6 +44,7 @@ import (
 
 const (
 	StorageDisk                  = "/mnt/?da1"
+	StorageDiskForGeneric        = "/"
```

Comment:
For the generic driver we can not assume that users would create machines with one `/` partition.  So I suggest we make it not applicable for generic driver. 

```
$ minishift status
Minishift:  Running
Profile:    minishift
OpenShift:  Running (openshift v3.9.0+b8265d6-31)
DiskUsage:  Not applicable (For generic VM driver)
CacheUsage: 4.33 GB (used by oc binary, ISO or cached images)
```
cc @minishift/minishift-dev

Reply:
For the generic driver we can not assume that users would create machines with one `/` partition.  So I suggest we make it not applicable for generic driver. 

```
$ minishift status
Minishift:  Running
Profile:    minishift
OpenShift:  Running (openshift v3.9.0+b8265d6-31)
DiskUsage:  Not applicable (For generic VM driver)
CacheUsage: 4.33 GB (used by oc binary, ISO or cached images)
```
cc @minishift/minishift-dev